grails-app/services/au/org/ala/ecodata/OrganisationService.groovy

```diff
@@ -217,4 +219,65 @@ class OrganisationService {
         organisationDetails
     }
 
+
+    /**
+     * Returns the reportable metrics for a organisation as determined by the organisation output targets and activities
+     * that have been undertaken.
+     * @param id identifies the organisation.
+     * @return a Map containing the aggregated results.
+     *
+     */
+    def organisationMetrics(String id, targetsOnly = false, approvedOnly = false, List scoreIds = null, Map aggregationConfig = null, boolean includeTargets = true) {
+        def org = Organisation.findByOrganisationId(id)
```

Comment:
Organisations don't have a concept of targets so we should delete targetsOnly and includeTargets